src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/UseNameofInPlaceOfString.cs

```diff
@@ -90,66 +88,49 @@ private void AnalyzeArgument(OperationAnalysisContext context)
             }
         }
 
-        private static IEnumerable<string> GetPropertiesInScope(OperationAnalysisContext context)
+        private static bool HasAPropertyMatchInScope(OperationAnalysisContext context, string stringText)
```

Comment:
[suggestion] Maybe `HasAnyPropertyMatchInScope` or `HasSomePropertyMatchInScope` to make it easier to read (the `AP` makes it weird to the eyes).